libmscore/articulation.cpp

```diff
@@ -270,6 +272,16 @@ void Articulation::read(XmlReader& e)
             else if (tag == "direction") {
                   setProperty(P_ID::DIRECTION, Ms::getProperty(P_ID::DIRECTION, e));
                   }
+            else if ( tag == "ornamentStyle") {
+                auto text = e.readElementText();
+                if ( text == "Baroque" || text == "1")
+                    setOrnamentStyle(MScore::OrnamentStyle::BAROQUE);
+                else
+                    setOrnamentStyle(MScore::OrnamentStyle::DEFAULT);
+            }
+            else if ( tag == "playArticulation")
```

Comment:
Hi Nicolas,
Jim originally had it working that way but I suggested making them independent to improve usability (see discussion here: https://musescore.org/en/node/56281#comment-263226). Basically, if you disable playback and later wish to re-enable it helps that the correct style is remembered for you, especially if there are multiple styles in use in a single score.